File: 1897-redistribute-characters-to-make-all-strings-equal/1897-redistribute-characters-to-make-all-strings-equal.cpp
class Solution {
public:
    bool makeEqual(vector<string>& words) {
        vector<int> cnt(26, 0);
        for(string word : words){
            for(char c : word) cnt[c-'a']++;
        }
        int n = words.size();
        for(int val : cnt){
            if(val % n != 0) return false;
        }
        return true;
    }
};

File: 1897-redistribute-characters-to-make-all-strings-equal/1897-redistribute-characters-to-make-all-strings-equal.java
class Solution {
    public boolean makeEqual(String[] words) {
        int[] cnt = new int[26];
        for(String word : words){
            for(Character c : word.toCharArray()) cnt[c-'a']++;
        }
        int n = words.length;
        for(int val : cnt){
            if(val % n != 0) return false;
        }
        return true;
    }
}